linera-service/tests/end_to_end_tests.rs

```diff
@@ -154,6 +151,7 @@ impl AmmApp {
     }
 }
 
+#[cfg_attr(feature = "", test_case(LocalNetConfig::new_test(Database::Service, Network::Grpc) ; "service_grpc"))]
```

Comment:
Does `feature = ""` do anything?

```suggestion
#[cfg(test_case(LocalNetConfig::new_test(Database::Service, Network::Grpc) ; "service_grpc")]
```

(Also below.)

Reply:
Yes, thank you for that.